package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/runtime": "^7.23.2",
```

Comment:
may be im wrong here,
but
as far as know -
babel is something that replaced by vite
and thus  - no more needed

Reply:
> maybe I'm wrong here, but as far as know - babel is something that replaced by vite and thus - no more needed

There was a conflict with the versions: 7.23.1 (my branch) // 7.23.2 (main). 
I don't think you are wrong. 
Could the yarn.lock file be different in different environments of the same project? How come my version was not updated when I run ```yarn install``` before starting to code? 

I had some bugs that I tried to resolve alone, but tell me if it caused bugs that I'm not aware of, please 🙏

Reply:
@amabelleS 
https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch
please take `package.json` and `yarn-lock.json` from `main`

Reply:
Please check this PR: 
https://github.com/hasadna/open-bus-map-search/pull/164

I opened a new branch on a new clone, fresh start with hopefully fewer bugs:)

Reply:
I am closing this PR because we have the same changes in #164